custom_components/ttlock/services.py

```diff
@@ -50,6 +52,32 @@ def register(self) -> None:
             ),
         )
 
+        self.hass.services.register(
+            DOMAIN,
+            "create_pass_code",
```

Comment:
In some places you have used `passcode` and in other places it's `pass_code`. Could we standardise on one for consistency?